src/node.cc

```diff
@@ -2809,8 +2793,8 @@ static void EnvQuery(Local<Name> property,
 #else  // _WIN32
     node::TwoByteValue key(info.GetIsolate(), property);
     WCHAR* key_ptr = reinterpret_cast<WCHAR*>(*key);
-    if (GetEnvironmentVariableW(key_ptr, nullptr, 0) > 0 ||
-        GetLastError() == ERROR_SUCCESS) {
+    GetEnvironmentVariableW(key_ptr, nullptr, 0);
+    if (GetLastError() != ERROR_ENVVAR_NOT_FOUND) {
```

Comment:
Is it safe to ignore the return value of `GetEnvironmentVariableW`?

Reply:
That was the working assumption at the time. But I just missed this part of the patch, I want to use `uv_os_getenv` here as well.

Reply:
`uv_os_getenv()` will return `UV_ENOENT` in this case instead of `ERROR_ENVVAR_NOT_FOUND`.